package.json

```diff
@@ -57,6 +57,7 @@
     "classnames": "^2.2.5",
     "coveralls": "^2.12.0",
     "css-loader": "^0.26.1",
+    "draft-convert": "^1.4.3",
```

Comment:
This should be in the dependencies and we shouldn't be importing the old library there